Style/style.css

```
@@ -141,3 +145,35 @@ h1 {
   line-height: 32px;
   margin-top: 25px;
 }
+
+@media screen and (max-width: 1200px) {
+  #header {
+    padding: 0;
+  }
+
+  #header-img {
+    margin-left: 24px;
+  }
+
+  #loginButton {
+    margin-right: 24px;
+  }
+
+  .item {
+    margin: 0 50px;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  #header-img {
+    margin: 24px;
+  }
+
+  #loginButton {
+    margin-right: 24px;
+  }
+
+  .item {
+    margin: 0 50px;
+  }
+}
```

❗️ 수정요청
해당 미디어 쿼리문을 보게되면 화면 사이즈가 768px인 경우 PC + Tablet에 작성된 스타일들을 상속받게 됩니다.
Tablet을 위한 조건문이 `@media (min-width: 769px) and (max-width: 1200px) {`였다면 화면 사이즈가 768px일때 해당 코드블럭 안의 스타일들은 상속되지 않았을거에요~
이러한 것을 인지하고 해당 스타일을 보면 불필요하게 중복되는 부분이 있으니 이를 염두에 두고 수정해보세요!

```suggestion
@media screen and (max-width: 1200px) {
  #header {
    padding: 0;
  }

  #header-img {
    margin-left: 24px;
  }

  #loginButton {
    margin-right: 24px;
  }

  .item {
    margin: 0 50px;
  }
}
```